### DISCUSSION TITLE
Extending time that splash screen is visible

### DISCUSSION THREAD

**Discussion by axtens** [unoplatform/uno#14577] [Q&A]
Thanks to @pictos we now have a splash screen and an icon. Not particularly sexy at this point but we're on the road. So the next question is, how do we extend the time that the splash screen appears as it is currently less than a second. There does not appear to be any property for it in the XML of base.props. Our base props now holds, in part, 
```
		<UnoSplashScreen
			Include="$(MSBuildThisFileDirectory)Splash\Peloton_Logo_MarkOnly_TransparentBack-lighter.svg"
			BaseSize="128,128"
			Color="red"  />
```
There appears to be no `Delay` keyword or the like.

**Comment by pictos** [comment]
@nickrandolph does the extended Splash solves this issue?
